filing_history_descriptions.yml

```diff
@@ -176,7 +176,7 @@ description:
     'liquidation-voluntary-arrangement-replacement-nominee-scotland' : "**Appointment of replacement nominee**"
     'liquidation-voluntary-arrangement-decision-to-extend-moratorium' : "**Decision to extend moratorium**"
     'liquidation-voluntary-arrangement-court-order-to-extend-moratorium' : "**Court order to extend moratorium**"
-    'liquidation-voluntary-arrangement-court-order': "**Court order in respect of a voluntary arrangement**"
+    'liquidation-voluntary-arrangement-or-moratorium-court-order' : "**Court order in respect of a voluntary arrangement or moratorium**"
```

Comment:
same comment as related chs-backend PR:

I believe this one is fine to update as its not currently live so not a breaking change, can you confirm this is the case?